alchy/utils.py

```diff
@@ -92,12 +92,12 @@ def get_mapper_class(model, field):
     return mapper_class(getattr(model, field))
 
 
-def merge_declarative_args(base_classes, config_key):
+def merge_declarative_args(cls, config_key):
     """Given a list of base classes, merge declarative args identified by
     `config_key` into a single configuration object.
     """
-    configs = [getattr(base, config_key, None)
-               for base in reversed(base_classes)]
+    configs = [base.__dict__.get(config_key)
+               for base in reversed(cls.__mro__)]
```

Comment:
Yeah, if there's a public method for accessing it, I'd rather do that than drop down to that layer.